ember-server/shared/src/main/scala/org/http4s/ember/server/internal/WebSocketHelpers.scala

```diff
@@ -196,7 +198,7 @@ object WebSocketHelpers {
         }
         Stream
           .iterable(chunks)
-          .flatMap(Stream.chunk(_))
+          .flatMap(Stream.chunk)
```

Comment:
Scala 3 requires the new syntax, I think fatal warnings will catch you here :)
```suggestion
          .flatMap(Stream.chunk(_))
```

Reply:
Huh, I must be confused 🤔  Ignore me.